team_project_access_integration_test.go

```diff
@@ -0,0 +1,281 @@
+package tfe
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestTeamProjectAccessesList(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	pTest, pTestCleanup := createProject(t, client, orgTest)
+	defer pTestCleanup()
+
+	tmTest1, tmTest1Cleanup := createTeam(t, client, orgTest)
+	defer tmTest1Cleanup()
+	tmTest2, tmTest2Cleanup := createTeam(t, client, orgTest)
+	defer tmTest2Cleanup()
+
+	tpaTest1, taTest1Cleanup := createTeamProjectAccess(t, client, tmTest1, pTest, orgTest)
+	defer taTest1Cleanup()
+	tpaTest2, taTest2Cleanup := createTeamProjectAccess(t, client, tmTest2, pTest, orgTest)
+	defer taTest2Cleanup()
+
+	t.Run("with valid options", func(t *testing.T) {
+		tpal, err := client.TeamProjectAccess.List(ctx, &TeamProjectAccessListOptions{
+			ProjectID: pTest.ID,
+			ListOptions: ListOptions{
+				PageNumber: 1,
+				PageSize:   100,
+			},
+		})
+		require.NoError(t, err)
+		assert.Contains(t, tpal.Items, tpaTest1)
+		assert.Contains(t, tpal.Items, tpaTest2)
+
+		//t.Skip("paging not supported yet in API")
```

Comment:
Is this still the case? I thought the API now supported pagination but I could easily be mistaken.